dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OpenApiIT.java

```diff
@@ -80,15 +85,32 @@ public static void afterClass() {
     @Disabled("no longer generated in 1.15 with transition to jakarta")
     void testSwagger20() {
         Response response = client.target(baseURL + "swagger.json").request().get();
-        assertThat(response.getStatus()).isEqualTo(HttpStatus.SC_OK);
+        assertEquals(HttpStatus.SC_OK, response.getStatus());
         // To prevent connection leak?
         response.readEntity(String.class);
     }
 
+    /*
+    Tests to make sure CORS headers are NOT present on a non-GA4GH endpoint
```

Comment:
Suggest adding a test that the CORS header is not on endpoints like https://dockstore.org/api/static/swagger-ui/index.html#/extendedGA4GH/Update%20the%20workflows%20and%20tools%20indices